packages/apollo-link-schema/src/schemaLink.ts

```diff
@@ -35,16 +35,17 @@ export class SchemaLink extends ApolloLink {
 
   public request(operation: Operation): Observable<FetchResult> | null {
     return new Observable<FetchResult>(observer => {
-      Promise.resolve(
-        execute(
+      new Promise(resolve => {
```

Comment:
@evenchange4 are you sure something is not wrong with your JavaScript environment, like are you using a weird polyfill for `Promise` or something?  [`Promise.resolve`[(http://devdocs.io/javascript/global_objects/promise/resolve) is now standard so I don't see why this change would be necessary, I think either your build or runtime environment isn't configured with a proper `Promise.resolve` implementation.

Reply:
You are right. There is some problem with my jest environment.

Reply:
Ah don't worry, I'm not a maintainer, I just happened to notice this.